python/hail/dataset.py

```diff
@@ -2199,29 +2199,43 @@ def ibd(self, maf=None, bounded=True, min=None, max=None):
 
     @handle_py4j
     @requireTGenotype
-    def ibd_prune(self, threshold, maf=None, bounded=True):
+    def ibd_prune(self, threshold, tiebreaking_expr=None, maf=None, bounded=True):
         """
         Prune samples from variant dataset based on PI_HAT values of IBD computation.
 
         **Examples**
 
-        Prune samples so that no two have a PI_HAT value greater than or equal to 0.4:
+        Prune samples so that no two have a PI_HAT value greater than or equal to 0.5:
 
-        >>> vds.ibd_prune(0.4)
+        >>> pruned_vds = vds.ibd_prune(0.5)
 
         **Notes**
 
         The variant dataset returned may change in near future as a result of algorithmic improvements. The current algorithm is very efficient on datasets with many small
-        families, less so on datasets with large families.
+        families, less so on datasets with large families. Currently, the algorithm works by deleting the person from each family who has the highest number of relatives,
+        and iterating until no two people have a PI_HAT value greater than that specified. If two people within a family have the same number of relatives, the tiebreaking_expr
```

Comment:
Only saw lowercase pi_hat once in IBD, and I decided to change it to uppercase since the annotation is stored as PI_HAT.